### DISCUSSION TITLE
怎么用命令修改默认端口号？

### DISCUSSION THREAD

**Discussion by chenmo-tssz** [IceWhaleTech/CasaOS#1204] [Q&A]
怎么用命令修改默认端口号？
安装成功了，发现和BT的80端口冲突，不知道怎么用命令修改默认端口号？有大神知道的吗？

**Comment by wangking6666** [comment]
I also want to know, but why hasn't anyone answered? Has the problem been resolved?

**Comment by junyiz** [comment]
试试修改 /etc/casaos/gateway.ini 里的 port
